modules/EvseManager/ErrorHandling.hpp

```diff
@@ -150,33 +64,28 @@ class ErrorHandling {
     void clear_powermeter_transaction_start_failed_error();
 
 private:
-    void raise_inoperative_error(const Everest::error::Error& error);
+    void process_error();
+    void raise_inoperative_error();
     void clear_inoperative_error();
+    bool errors_prevent_charging();
+
+    // p_evse. We need to ignore Inoperative here as this is the result of this check.
+    const std::list<Everest::error::ErrorType> ignore_errors_evse{"evse_manager/Inoperative"};
+    const std::list<Everest::error::ErrorType> ignore_errors_bsp{"evse_board_support/MREC3HighTemperature",
+                                                                 "evse_board_support/MREC18CableOverTempDerate",
+                                                                 "evse_board_support/VendorWarning"};
+    const std::list<Everest::error::ErrorType> ignore_errors_connector_lock{"connector_lock/VendorWarning"};
+    const std::list<Everest::error::ErrorType> ignore_errors_ac_rcd{"ac_rcd/VendorWarning"};
+    const std::list<Everest::error::ErrorType> ignore_errors_imd{"isolation_monitor/VendorWarning"};
+    const std::list<Everest::error::ErrorType> ignore_errors_powersupply{"power_supply_DC/VendorWarning"};
```

Comment:
these don't necessarily need to be defined inside the class, they could as well live outside.  If you're unlucky, each instance might allocate them.  When having 6 variables beginning with `ignore_errors_*`, it might be better to have a struct like
```
struct IgnoreErrors {
  using ErrorList = std::list<Everest::error::ErrorType>;
  ErrorList evse, bsp, connector_lock ....;
};
```
And then initialize this once as a static const like
```
static const IgnoreErrors ignore_errors {
 {"evse_manager/Inoperative"}, /* evse */
 {"evse_board_support ....", " .. " }, /* bsp */
};
```
or similar

Reply:
Doin the initialization like this is quite easy to get wrong, especially there is no real typing here and it is all strings. It is quite easy to get the order wrong when adding new things and it will compile fine etc.

changed it to this to avoid the separate decl/instanc:

```
using ErrorList = std::list<Everest::error::ErrorType>;
static const struct IgnoreErrors {
    // p_evse. We need to ignore Inoperative here as this is the result of this check.
    ErrorList evse{"evse_manager/Inoperative"};
    ErrorList bsp{"evse_board_support/MREC3HighTemperature", "evse_board_support/MREC18CableOverTempDerate",
                  "evse_board_support/VendorWarning"};
    ErrorList connector_lock{"connector_lock/VendorWarning"};
    ErrorList ac_rcd{"ac_rcd/VendorWarning"};
    ErrorList imd{"isolation_monitor/VendorWarning"};
    ErrorList powersupply{"power_supply_DC/VendorWarning"};
} ignore_errors;
```

Reply:
Yes, that is a tradeoff.  To solve that problem you could do
```
struct IgnoreErrors {
  static const ErrorList evse;
  static const ErrorList ...
};

const ErrorList IgnoreErrors::evse = {" .... /.."};
```
This also has the advantage that you don't need to instantiate the struct and can just use `IgnoreErros::evse`

Reply:
Or just simply us a namespace